src/components/calculator-component.js

```diff
@@ -0,0 +1,11 @@
+function calculator(bodyId,screenId,inputId,buttonsGroup) {
```

Comment:
why bodyId?

Reply:
because it the id of the body calc
do you think it's better **bodyCalcId**?

Reply:
because bodyId refers to the id of the whole body of the calculator

Reply:
but if you have a `id` in calculator, doesn't it mean the calculator's `id`? 👀

Reply:
hmm makes sense

Reply:
ok, now it's just id on